deploy/charts/harvester/values.yaml

```diff
@@ -450,3 +450,8 @@ loop-device-cleaner:
     imagePullPolicy: IfNotPresent
     repository: alpine
     tag: 3.12
+
+syncAdditionalCA:
```

Comment:
Thanks for the update. Do you think there is a need for a more general field name that can be re-used by jobs. For example, there is a workaround daemonset to remove stale longhorn loop devices. Just feel this kind of job can share the same image.

Reply:
Yes, I update it. Thank you.